daml-script/runner/src/main/scala/com/digitalasset/daml/lf/engine/script/Converter.scala

```diff
@@ -209,29 +210,31 @@ object Converter {
       case _ => Left(s"Expected CreateAndExercise but got $v")
     }
 
+  private[this] val tupleFieldNames =
```

Comment:
Is the idea that this is supposed to be sorted? If so, we should document this in a comment.

Reply:
It is still not sorted. 
I just want the names and the indexes of the field factored out, so it simplify changes when I make the fields sorted. 
I will change the type of `SStruct#fiedls` in an upcoming PR.

Reply:
But the indices will be wrong if `tupleFieldNames` is not sorted right?

Reply:
Currently, the indices should be in the same order as you build the arguments of the `SBStructCon` builtin.
Latter `SBStructCon` will reorder the field, only then the order will be relevant.

I replace the line 
```
 private[this] val Seq(fstIdx, sndIdx) = tupleFieldNames.indices
```
by 
```
  private[this] val fstIdx = 0
  private[this] val sndIdx = 1
``` 
So now, the change really just extract some expressions as values (+ renaming `"a"` to `"fst"` and `"b"` to `"second"`)

Reply:
Thanks for the explanation!